### ISSUE TITLE
IPFS error: QuotaExceededError 

### ISSUE THREAD

**Issue by geckoslair** [Paratii-Video/paratii-js#292]
Getting this on my Ubuntu 16.04 machine.
Chrome Version 64.0.3282.186 (Official Build) (64-bit)
 
[portal.paratii.video-1530634993118.log](https://github.com/Paratii-Video/paratii-js/files/2160198/portal.paratii.video-1530634993118.log)

**Comment by jellegerbrandy** [comment]
So I assume this is because the storage spac eis full. Am I right, @ya7ya ?

**Comment by ya7ya** [comment]
@jellegerbrandy yeah, it's because his client side browser storage is full.

**Comment by jellegerbrandy** [comment]
Is this because all files that you upload are pinned?
so what do we do in these cases? Do we do an automatic cleanup?

**Comment by ya7ya** [comment]
No, this is mainly because `js-ipfs` doesn't have garbage collection yet. and we also change the repo path with `Math.random()` during testing alot. which means there are alotta `ipfs-repos` that have duplicated data at this point. 

I made sure that we use the same `ipfs-repo` during production. and hopefully garbage collection will be implemented soon since the pinning is finally ready and released in `ipfs@0.30.0`
